src/utils/node/mergeConfigs.js

```diff
@@ -187,6 +187,42 @@ export function createMergeConfigsFunction(cx) {
   return mergeConfigs;
 }
 
+export function createGetMergedConfigFunction(cx) {
```

Comment:
`createGetMergedConfigFunction` > `createConfigMerge` (use more short and clear name)